tdp/cli/commands/init.py

```diff
@@ -16,7 +16,8 @@
     envvar="TDP_OVERRIDES",
     required=False,
     type=click.Path(exists=True, resolve_path=True, path_type=Path),
-    help="Path to tdp vars overrides",
+    multiple=True,
```

Comment:
I think we should have the same behavior as `TDP_COLLECTION_PATH`, use the OS path separator to have multiple path in order to specify multiple path with `TDP_OVERRIDES` env variable.

Reply:
I showed in the description of this PR that with this solution, `TDP_OVERRIDES` can accept multiple paths and behaves exactly like `TDP_COLLECTION_PATH` with OS separator, as it is natively supported by Click. Here is what Click's documentation states :
> The default implementation for all types is to split on whitespace. The exceptions to this rule are the [File](https://click.palletsprojects.com/en/8.1.x/api/#click.File) and [Path](https://click.palletsprojects.com/en/8.1.x/api/#click.Path) types which both split according to the operating system’s path splitting rules. On Unix systems like Linux and OS X, the splitting happens for those on every colon (:), and for Windows, on every semicolon (;).

See documentation and example here : https://click.palletsprojects.com/en/8.1.x/options/#multiple-values-from-environment-values.

I thought it was the simplest way to allow multiple paths, letting Click do the platform independent parsing. In fact,
 `TDP_COLLECTION_PATH` (`--collection-path`) could benefit from this too.

Let me know what you think, and I'll make change to this PR accordingly. Thanks for your review.

Reply:
Ok to keep `multiple=true` if this the standard way with Click. I open #303 to change `--collection-path` behavior to match the new one.